enterprise/frontend/src/embedding-sdk-bundle/components/public/ComponentProvider/ComponentProvider.tsx

```diff
@@ -31,6 +32,24 @@ type ComponentProviderInternalProps = ComponentProviderProps & {
   isLocalHost?: boolean;
 };
 
+let hasInitializedPlugins = false;
```

Comment:
I wonder if this being a global value can cause any issues with testing plugins. If it does we could move it into the hook.

Reply:
No it's fine as far as I can tell